src/GeneralStateTestsFiller/stShift/shl10Filler.json

```diff
@@ -0,0 +1,107 @@
+{
+    "shl10" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : ["<=Byzantium"],
+                "result" : {
+                    "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                        "balance" : "1000000000000000000",
+                        "code" : "0x600160001b600055",
+                        "storage" : {
+                            "0x00" : "0x00"
+                        }
+                    },
+                    "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
+                        "balance" : "400000",
+                        "code" : "0x",
+                        "nonce" : "0",
+                        "storage" : {
+                        }
+                    },
+                    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                        "balance" : "999999999999600000",
+                        "code" : "0x",
+                        "nonce" : "1",
+                        "storage" : {
+                        }
+                    }
+                }
+            },
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : [">=Constantinople"],
+                "result" : {
+                    "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                        "balance" : "1000000000000000000",
+                        "code" : "0x600160001b600055",
+                        "storage" : {
+                            "0x00" : "0x00"
+                        }
+                    },
+                    "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
+                        "balance" : "400000",
+                        "code" : "0x",
+                        "nonce" : "0",
+                        "storage" : {
+                        }
+                    },
+                    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                        "balance" : "999999999999600000",
+                        "code" : "0x",
+                        "nonce" : "1",
+                        "storage" : {
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                "balance" : "1000000000000000000",
+                "code" : "0x600160001b600055",
```

Comment:
If possible it might be nice to have test fillers in a separate commit

Reply:
The code is without comment

Reply:
Added.